packages/wonder-blocks-popover/components/popover.test.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import * as React from "react";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+import Popover from "./popover.js";
+import PopoverContent from "./popover-content.js";
+import {PopoverContentCore} from "../index.js";
+
+describe("Popover", () => {
+    afterEach(() => {
+        unmountAll();
+    });
+
+    it("should set the anchor as the popover ref", () => {
+        // Arrange
+        const wrapper = mount(
+            <Popover
+                placement="top"
+                content={<PopoverContent title="Title" content="content" />}
+            >
+                {({open}) => (
+                    <button data-anchor onClick={open}>
+                        Open default popover
+                    </button>
+                )}
+            </Popover>,
+        );
+
+        // Act
+        const anchorElement = wrapper.find("[data-anchor]").getDOMNode();
+
+        // Assert
+        expect(wrapper.state("anchorElement")).toBe(anchorElement);
+
+        expect(wrapper.find(PopoverContent)).toHaveLength(0);
```

Comment:
Maybe we can change these expect statements to be something like `expect(wrapper.find(PopoverContent).exists()).toBeFalsy();` (or `toBeTruthy()`). I think it’s a little clearer than checking the length to be 0 or 1 to see if it exists.

Reply:
Good call, we can also use a jest-enzyme matcher for that :). I'll update that.

https://www.npmjs.com/package/jest-enzyme#toexist